index.js

```diff
@@ -60,6 +60,9 @@ var Sharp = function(input) {
     gamma: 0,
     greyscale: false,
     normalize: 0,
+    // overlay
+    hasOverlay: false,
```

Comment:
@lovell: If you show me how to test check if a string is empty in C++, we might be able to get rid of `hasOverlay`. On the other hand, having this flag might be more understandable instead of just relying on empty string `overlayPath`.